GAPS (#47)
msupply-foundation/msupply_docs: content/docs/programs/immunizations.md

+++
title = "Immunizations"
description = "Immunization Programs"
date = 2022-05-17T16:20:00+00:00
updated = 2022-05-17T16:20:00+00:00
draft = false
weight = 62
sort_by = "weight"
template = "docs/page.html"

[extra]
toc = true
top = false
+++

> Choose `Programs` > `Immunizations` in the navigation panel.

Reviewer: this is highlighting the fact that `patient programs` and `Requisitions` belong in the `programs` menu and are absent!

You will be presented with a list of Immunization Programs:

![immunizations list](/docs/programs/images/immunizations.png)

The list of Immunization Programs is divided into 2 columns:

| Column              | Description                                                      |
| :------------------ | :--------------------------------------------------------------- |
| **Program Name**    | The name of the Immunization Program                             |
| **Vaccine Courses** | The Vaccine Courses that are a part of that Immunization Program |

## Adding a new Immunization Program

To add a new Immunization Program, click the `New Program` button in the top right.

![immunizations: new program button](/docs/programs/images/immunizations_new_program_button.png)

This will open a new window, where you can enter the name of the new Immunization Program. Note that all programs must have a unique name.

![immunizations: new program modal](/docs/programs/images/immunizations_new_program_modal.png)

- Click `OK` once you are happy with your changes
  - You will be redirected to the detail view for your new Immunization Program

> - OR: Click `Cancel` at any time to revert your changes

Reviewer: ```suggestion
-Click `Cancel` to close the window if you've changed your mind
```

or something? there's not much to revert so that didn't sound right..

![vaccine courses: select to delete](/docs/programs/images/vaccine_courses_select.png)

Open the `Actions` dropdown in the top right corner, above the list, and select `Delete selected lines`.

![vaccine courses: actions](/docs/programs/images/immunizations_actions.png)

You will be asked to confirm that you wish to delete the selected Vaccine Courses. Press `Cancel` to prevent deleting, or `Ok` to confirm.

![vaccine courses: confirm delete](/docs/programs/images/vaccine_courses_confirm_delete.png)

A notification in the bottom left corner will confirm how many courses were deleted.

## Viewing Vaccine Course Details

To view a Vaccine Course in more detail, click on it. This will take you to the detail page for that course.

![vaccine course: detail](/docs/programs/images/vaccine_course_detail.png)

You can use the `Close` button in the bottom right to close the detail view, and go back to the list of Vaccine Courses within this Immunization Program.

> From this view, you can edit all the relevant information for the Vaccine Course, including its name, details like its wastage rate, and the number of doses.

Author: Maybe its worth adding here, demographics are configured in demographics page, vaccine items are from mSupply?? wasn't sure how much this page would change before adding much detail lol
